src/Utility/AnaSol.h

```diff
@@ -46,24 +47,25 @@ class AnaSol
   //! allocated objects, as the class destructor will attempt to delete them.
   AnaSol(RealFunc* s1 = 0, VecFunc* s2 = 0,
          VecFunc* v1 = 0, TensorFunc* v2 = 0, STensorFunc* v3 = 0)
-    : scalSol(s1), scalSecSol(s2), vecSol(v1), vecSecSol(v2), stressSol(v3) {}
+    : scalSol{s1}, scalSecSol{s2}, vecSol(v1),
+      vecSecSol(v2), stressSol(v3) {}
 
   //! \brief Constructor initializing the primary and secondary solution fields.
   //! \param[in] s Primary scalar solution field
   //! \param[in] sigma Symmetric stress tensor field
   AnaSol(RealFunc* s, STensorFunc* sigma)
-    : scalSol(s), scalSecSol(0), vecSol(0), vecSecSol(0), stressSol(sigma) {}
+    : vecSol(0), vecSecSol(0), stressSol(sigma) {}
```

Comment:
Missing initialization of scalSol here, need to be consistent: `{ if (s) scalSol.push_back(s); }`